src/lib/sdk.ts

```diff
@@ -241,6 +241,20 @@ export default class Strapi {
     });
   }
 
+  /**
+   * Get the total count of entries with the provided criteria
+   * @param contentTypePluralized
+   * @param params Filter and order queries.
+   */
+  public getEntryCount(
+    contentTypePluralized: string,
```

Comment:
Could you change all `contentTypePluralized` occurrence to `contentType`.

Reply:
Yes sir. Fixing now.

Reply:
It is done.